src/chrome/content/rules/Luadns.com.xml

```diff
@@ -0,0 +1,19 @@
+<!--
+	Nonfunctional hosts in *.luadns.com:
+		- status.luadns.com
+		- uptime.luadns.com
```

Comment:
@J0WI, I think you forgot to explain what the problem is for these hosts.

Reply:
Yes :laughing: